CHANGELOG.md

```diff
@@ -733,9 +733,9 @@ Patch release, resolves two issues:
 
 * Pub
 
-  * Avoid using a barback asset server for executables unless they actually use
-    transformers. This makes precompilation substantially faster, produces
-    better error messages when precompilation fails, and allows
+  * Avoid using a bareback asset server for executables unless they actually use
```

Comment:
"Barback" is actually correct. It refers to https://github.com/dart-lang/barback

(Otherwise fine).